tools/wave/network/api/api_handler.py

```diff
@@ -68,29 +68,29 @@ def create_hal_list(self, items, uris, index, count, total):
 
             if "self" in uris:
                 self_uri = uris["self"]
-                self_uri += f"?index={index}&count={count}"
+                self_uri += "?index={}&count={}".format(index, count)
```

Comment:
It's unclear to me why this is being changed away from f-strings? This is all supported from Python 3.6 onwards.